scopes/lanes/diff/lane-diff.cmd.ts

```diff
@@ -3,17 +3,33 @@ import { ScopeMain } from '@teambit/scope';
 import { Workspace } from '@teambit/workspace';
 import chalk from 'chalk';
 import { outputDiffResults } from '@teambit/legacy/dist/consumer/component-ops/components-diff';
+import { COMPONENT_PATTERN_HELP } from '@teambit/legacy/dist/constants';
 import { LaneDiffGenerator } from './lane-diff-generator';
 
 export class LaneDiffCmd implements Command {
-  name = 'diff [values...]';
+  name = 'diff [from] [to]';
```

Comment:
I really wanted to do it this way myself :) what hold me back is the case you pass only one param, which becomes `bit lane diff [to]`. Not `bit lane diff [from]`.  I don't have a good solution for this..

Reply:
how about 'diff [from/to] [to]'

Reply:
I just tried it locally, I can't put `/` inside an arg name

Reply:
so how about 'diff [from-or-to] [to]'

Reply:
It's better. But still, I think it'll be cumbersome to refer to this value when writing examples. I'm not sure it's better than what we have now.
Anyway, changing this, it's not only about the `name` property, it needs to be change in many places, because now instead of expecting an array of "values" we expect two optional named params, so it's probably out of scope of this PR.

Reply:
ok, i'll revert and relying on you to think of something at some point :)